ebpf-io/ebpf.io-website
# feat: add japanese localization

**Describe what changes this pull request brings**

Sections with translation:

- [x] Footer
- [x] SEO

Pages with translation:

- [ ] [What is eBPF?]()
- [ ] [Get Started]()
- [ ] [Labs]()
- [ ] [Applications]()
- [ ] [Contribute]()

**Steps to check:**

1. Open [preview]()
2. Make sure that everything looks and works as expected
